scripts/lib/ts-generator.js

```diff
@@ -300,8 +333,10 @@ TSGenerator.prototype.generateTypingsFromShape = function generateTypingsFromSha
         code += tabs(tabCount) + 'export type ' + shapeKey + ' = Date;\n';
     } else if (type === 'boolean') {
         code += tabs(tabCount) + 'export type ' + shapeKey + ' = boolean;\n';
-    } else if (type === 'blob' || type === 'binary') {
-        code += tabs(tabCount) + 'export type ' + shapeKey + ' = Buffer|Uint8Array|Blob|string;\n';
+    } else if (type === 'blob' || type === 'binary') {     
+        code += tabs(tabCount) + 'export type ' + shapeKey + ' = Buffer|Uint8Array|Blob|string'
+        + self.addReadableType(shapeKey)
```

Comment:
Style nit: continuation lines (lines that are part of the same statement as the preceding line) should be indented.